src/toil/lib/docker.py

```diff
@@ -0,0 +1,255 @@
+import base64
+import logging
+import subprocess
+
+import os
```

Comment:
Pycharm's optimize imports put it there, not sure why.  I agree it makes sense to move it among the core library imports.